src/utils/path_helpers.py

```diff
@@ -95,8 +95,9 @@ def create_staging_config(config: dict) -> dict:
     staging_dict[
         "pnp_staging_qa_path"
     ] = f"{paths['pnp_path']}{config['pnp_paths']['staging_qa_path']}"
-    staging_dict["manual_outliers_path"] = f"{berd_path}{paths['manual_outliers_path']}"
-    staging_dict["manual_imp_trim_path"] = f"{berd_path}{paths['manual_imp_trim_path']}"
+
+    staging_dict["manual_outliers_path"] = f"{paths['manual_outliers_path']}"
+    staging_dict["manual_imp_trim_path"] = f"{paths['manual_imp_trim_path']}"
```

Comment:
@zorge69 I don't think this is correct, if we make this change, then we can't read in the file. Did you make these changes in the package directly in DAP? What we get here is just "'06_imputation/manual_trimming/2023_manual_trimming_v1.csv'" not the full path.